janusgraph-solr/src/test/resources/docker-compose.yml

```diff
@@ -23,7 +23,7 @@ services:
       - "3888:3888"
 
   solr:
-    image: solr:5.5.4
+    image: solr:7.7
```

Comment:
There is Solr 7.7.1 release available already.

Reply:
Yes, but 7.7 points to the same image as 7.7.1 right now. If however 7.7.2 gets released, then 7.7 will point to that which means that it will be more up to date.

Reply:
Ok